server/src/index.ts

```diff
@@ -0,0 +1,31 @@
+import express, { Express, Request, Response } from 'express'
+import helmet from 'helmet'
+import cors from 'cors'
+import connectedDB from  './config/connectionDB'
+
+const port = process.env.PORT || 3040
+const app = express()
+
+// Middlewares
+app.use(cors())
+app.use(helmet())
+
+
+app.get('/', (req: Request, res: Response) => {
+	res.send
+('Hiii!')
```

Comment:
for testing?

Reply:
Yes, it was for testing, I did remove it